GitUI/Interops/User32/GetWindowDC.cs

```diff
@@ -7,5 +7,8 @@ internal static partial class NativeMethods
     {
         [DllImport(Libraries.User32)]
         public static extern IntPtr GetWindowDC(IntPtr hwnd);
+
+        [DllImport(Libraries.User32)]
+        public static extern void ReleaseDC(IntPtr hwnd, IntPtr dc);
```

Comment:
Move it to GitUI/Interops/User32/ReleaseDC.cs

Reply:
Yes, of course.